app/boards/arm/cyber60/cyber60.dts

```diff
@@ -0,0 +1,151 @@
+			label = "Red LED";
+		};
+		green_led: led_2 {
+			gpios = <&gpio0 28 GPIO_ACTIVE_LOW>;
+			label = "Green LED";
+		};
+		blue_led: led_1 { // also broken out on header con307 on cyber60-2 Rev A1
+			gpios = <&gpio0 5 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+	};
+
+	vbatt {
+		compatible = "zmk,battery-voltage-divider";
+		label = "BATTERY";
+		io-channels = <&adc 1>;
+		output-ohms = <100000>;
+		full-ohms = <(100000 + 100000)>;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <11>;
+		rows = <6>;
+		map = <
+RC(0,0) RC(0,1) RC(1,1) RC(0,2) RC(0,3) RC(0,4) RC(1,4) RC(0,5) RC(0,6) RC(0,7) RC(1,7) RC(0,8) RC(0,9) RC(0,10) RC(1,10)
+RC(1,0)   RC(2,1) RC(1,2) RC(2,2) RC(1,3) RC(2,4) RC(1,5) RC(1,6) RC(2,6) RC(2,7) RC(1,8) RC(1,9) RC(2,9)   RC(3,10)
+RC(2,0)     RC(3,1) RC(3,2) RC(2,3) RC(3,3) RC(3,4) RC(2,5) RC(3,6) RC(4,6) RC(3,7) RC(2,8) RC(3,9)          RC(2,10)
+RC(3,0)       RC(4,0) RC(4,1) RC(4,2) RC(4,3) RC(4,4) RC(3,5) RC(4,5) RC(5,6) RC(4,7)    RC(4,8)     RC(4,8) RC(4,9) RC(4,10)
+RC(5,0)   RC(5,1)   RC(5,2)  RC(5,3)                    RC(5,4)                     RC(5,5) RC(5,7) RC(5,8) RC(5,9) RC(5,10)
+		>;
+	};
+
+	kscan0: kscan {
+		compatible = "zmk,kscan-gpio-matrix";
+		label = "KSCAN";
+
+		diode-direction = "col2row";
+		row-gpios
+			= <&gpio0 7  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 23 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio1 9 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio0 12 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&gpio1 11 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+		col-gpios
+			= <&gpio0 2  GPIO_ACTIVE_HIGH>
+			, <&gpio1 10  GPIO_ACTIVE_HIGH>
+			, <&gpio0 4  GPIO_ACTIVE_HIGH>
+			, <&gpio0 19 GPIO_ACTIVE_HIGH>
+			, <&gpio1 4 GPIO_ACTIVE_HIGH>
+			, <&gpio0 22  GPIO_ACTIVE_HIGH>
+			, <&gpio1 6 GPIO_ACTIVE_HIGH>
+			, <&gpio1 2  GPIO_ACTIVE_HIGH>
+			, <&gpio1 1  GPIO_ACTIVE_HIGH>
+			, <&gpio1 3  GPIO_ACTIVE_HIGH>
+			, <&gpio1 0  GPIO_ACTIVE_HIGH>
+			;
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&adc {
+	status = "okay";
+};
+
+&usbd {
+	status = "okay";
+};
+
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "adafruit_boot";
+			reg = <0x000000000 0x0000C000>;
+		};
+		code_partition: partition@26000 {
+			label = "code_partition";
+			reg = <0x00026000 0x000d2000>;
+		};
+
+		/*
+		 * The flash starting at 0x000f8000 and ending at
+		 * 0x000fffff is reserved for use by the application.
+		 */
+
+		/*
+		 * Storage partition will be used by FCB/LittleFS/NVS
+		 * if enabled.
+		 */
+		storage_partition: partition@f8000 {
+			label = "storage";
+			reg = <0x000f8000 0x00008000>;
+		};
+	};
+};
```

Comment:
It looks like you're using the Adafruit bootloader. This set up will cause the board to corrupt after a while. Please take a look at the updated flash set up in #322

Reply:
Great! I changed to the updated flash set up.